### PR TITLE
Replaced old twitter icon with new icon

### LINKED ISSUE #1112: Replace old twitter icon with new icon
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Context

On the [website](https://virtualcoffee.io/) under navbar section the twitter logo is still the old one.
It needs to be updated.

### Proposed solution

Replacing the old twitter logo with new one. 

### Alternatives Considered

_No response_

### Additional Resources

_No response_

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

### PR DESCRIPTION
**Pull request by antriksh-9**
## Linked Issue

Closes #1112.

## Description

On the [website](https://virtualcoffee.io/) under navbar section the twitter logo was old one.
I changed it to the new logo.

Screenshot 2024-01-30 230417

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: app/components/Nav.tsx [hunk 1]
```diff
@@ -80,14 +80,13 @@ export default function Nav() {
...
 						>
 							<svg
-								role="img"
-								viewBox="0 0 24 24"
-								height="24"
-								width="24"
 								xmlns="http://www.w3.org/2000/svg"
+								width="24"
+								height="24"
+								viewBox="0 0 512 512"
```
Comment (adiati98) [inline app/components/Nav.tsx:86]: Hi @antriksh-9,
Is there any reason to remove the `role="img"` attribute and change the `viewBox` dimension?

File: app/components/Nav.tsx [hunk 2]
```diff
@@ -80,14 +80,13 @@ export default function Nav() {
...
 							>
 								<title>Twitter</title>
```
Comment (adiati98) [inline app/components/Nav.tsx:88]: You might also want to change this from "Twitter" to "X".

```suggestion
								<title>X</title>
```

**Comment by antriksh-9** [comment]
Hey @adiati98 I have made the necessary changes.

File: app/components/Nav.tsx [hunk 3]
```diff
@@ -82,12 +82,12 @@ export default function Nav() {
 							<svg
 								role="img"
 								viewBox="0 0 24 24"
```
Comment (danieltott) [inline app/components/Nav.tsx:84]: Looks like we needed that viewbox change - not sure what the problem there was...

If you update that we can get this up!

```suggestion
								viewBox="0 0 512 512"
```

**Review by danieltott — CHANGES_REQUESTED** [review]
Check my comment - we definitely needed the viewbox change - now the icon isn't showing up!

image

**Review by danieltott — APPROVED** [review]
Looks good - thanks @antriksh-9!

**Pull request merged**